source/_integrations/tautulli.markdown

```diff
@@ -3,16 +3,26 @@ title: Tautulli
 description: Instructions on how to set up Tautulli sensors in Home Assistant.
 ha_category:
   - Sensor
+ha_config_flow: true
 ha_release: 0.82
 ha_iot_class: Local Polling
 ha_codeowners:
   - '@ludeeus'
+  - '@tkdrob'
 ha_domain: tautulli
 ha_platforms:
   - sensor
 ---
 
-The `tautulli` sensor platform will monitor activity on a given [Tautulli Server][tautulli]. It will create a sensor that shows the number of currently active streams as the state. If you click the sensor for more details it will show you more statistics, the following stats are available by default:
+The Tautulli integration will monitor activity on a given [Tautulli Server][tautulli]. It will create a service with a sensor that shows the number of currently active streams as the state. If you click the sensor for more details, it will show you more statistics.
```

Comment:
```suggestion
The Tautulli integration will monitor activity on a given [Tautulli Server][tautulli]. It will create a sensor that shows the number of currently active streams as the state. If you click the sensor for more details, it will show you more statistics.
```

Using "service" in this context is confusing to the reader, the device type is service, but it's not a service like they are used to /dev-tools/services